Functions/Quantification/xASL_quant_FSL.m

```diff
@@ -450,13 +557,20 @@
 		end
 
 		% Print labeling durations
-		if x.modules.asl.bMultiPLD
-			for iLabDurs = 1:length(LabDurs)
-				fprintf(FIDoptionFile, '--tau%d=%.2f\n', iLabDurs, LabDurs(iLabDurs));
-			end
-		else
-			fprintf(FIDoptionFile, '--tau=%.2f\n', LabDurs);
-		end
+        if x.modules.asl.bMultiPLD
+            for iLabDurs = 1:length(LabDurs)
```

Comment:
I think this won't work nicely for merging three sessions - will fix that later.

Reply:
You can close this. But I would generalize the session merging such that it iterates over all sessions defined in sessionMergingList (unless you have a good reason to do this in #1543).